app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,10 @@
             android:exported="false" />
         <activity
             android:name=".AccountActivity"
+            android:parentActivityName=".MainSettingsActivity"
+            android:exported="false"/>
+      
+
             android:exported="false"
             android:parentActivityName=".MainActivity" />
```

Comment:
Must by cleaned

Reply:
in which way ?

Reply:
android:exported="false"
android:parentActivityName=".MainActivity" />
I don't know where these two lines come from, but think they are useless. Try to remove them and same for the useless spaces.

Reply:
It is maybe a residue of something that has been deleted